cmd/processor/app/processor.go

```diff
@@ -174,6 +178,28 @@ func NewProcessor(configurationPath string, platformConfigurationPath string) (*
 
 // Start starts the processor
 func (p *Processor) Start() error {
+
+	// create a goroutine that restarts a trigger if needed
+	go func() {
+		for {
+			select {
+			case triggerInstance := <-p.restartTriggerChan:
+
+				p.logger.DebugWith("Restarting trigger", "triggerInstance", triggerInstance)
```

Comment:
```suggestion
				p.logger.WarnWith("Restarting trigger", "triggerInstance", triggerInstance)
```

Reply:
imo instead of logging the entire `triggerInstance`, simply log the trigger name/kind.